ec2/sc-ec2-linux-jumpcloud-notebook-v1.0.0.yaml

```diff
@@ -433,6 +435,10 @@ Resources:
           Value: !Sub "Service Catalog instance created by ${AWS::StackName}"
         - Key: "parkmycloud"
           Value: "yes"
+        - Key: "ManagedInstanceMaintenanceTarget"
```

Comment:
could you please tell us what is the purpose of these tags?  how do you plan to use them?

Also should we apply these same tags to the templates in Sage-Bionetworks/aws-infra?

Reply:
ManagedInstanceMaintenanceTarget is a boolean for whether we will manage it centrally or not. PatchGroup is a flag that the scheduled tasks can use to determine when and which patches to apply. For now, let's just use prod-default for SC, but later we may also want test-default, prod-updateall, prod-nopatch, etc.